app/kontainerdriver_data.go

```diff
@@ -88,6 +88,16 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 	); err != nil {
 		return err
 	}
+	if err := creator.addCustomDriver(
+		"oraclecontainerengine",
+		"https://github.com/rancher-plugins/kontainer-engine-driver-oke/releases/download/v1.3.0/kontainer-engine-driver-oke-linux",
+		"",
```

Comment:
A checksum needs to be added, this is how Rancher knows we have downloaded the correct driver and not an imposter.

Reply:
Ill add a checksum.